cms/djangoapps/contentstore/views/tests/test_course_updates.py

```diff
@@ -34,6 +39,9 @@ def get_response(content, date):
 
             return json.loads(resp.content)
 
+        # initialize the Notification subsystem
+        startup.startup_notification_subsystem()
```

Comment:
I see you are calling this in a few places. Should we just move this to be in the setUp() method?